From 4afc63270ac388e5c2ffaf5fdb2bee6c6dc86d04 Mon Sep 17 00:00:00 2001
From: Emil Medve <Emilian.Medve@Freescale.com>
Date: Sat, 14 Feb 2015 02:52:20 -0600
Subject: [PATCH 45/54] iommu/fsl: PAMU is also present on 64-bit SoC(s)

This reverts a change done in '477ab7a iommu: Make more
drivers depend on COMPILE_TEST'

Signed-off-by: Emil Medve <Emilian.Medve@Freescale.com>
Signed-off-by: Joerg Roedel <jroedel@suse.de>
---
 drivers/iommu/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/iommu/Kconfig b/drivers/iommu/Kconfig
index 1ae4e54..33b6345 100644
--- a/drivers/iommu/Kconfig
+++ b/drivers/iommu/Kconfig
@@ -50,7 +50,6 @@ config OF_IOMMU
 
 config FSL_PAMU
 	bool "Freescale IOMMU support"
-	depends on PPC32
 	depends on PPC_E500MC || COMPILE_TEST
 	select IOMMU_API
 	select GENERIC_ALLOCATOR
-- 
2.1.0.27.g96db324

